test/framework.cc

```diff
@@ -33,6 +33,11 @@ uint64_t TestContext::getMonotonicTimeNanoseconds() {
   unimplemented();
   return 0;
 }
+proxy_wasm::WasmResult TestContext::log(uint32_t log_level,
+                                        std::string_view message) {
+  std::cout << "LOG from testcontext: " << message << std::endl;
```

Comment:
Did you intend to leave this in here? Fine if so, just wanted to call attention in case it was not

Reply:
Yep thanks, not really Rust specific but I found myself stashing and reinstating this over and over while working on plugins.